lib/components/user/user-account-screen.js

```diff
@@ -131,28 +195,30 @@ class UserAccountScreen extends Component {
             // can be wired to be managed by Formik.
             props => {
```

Comment:
Can you rename this to be `formikProps`?

Reply:
Updated in https://github.com/opentripplanner/otp-react-redux/pull/224/commits/b1d3d7c06753a056c14bdee4c5289b222462a7e6.